lib/cppcheck.h

```diff
@@ -203,7 +203,7 @@ class CPPCHECKLIB CppCheck {
      * @brief Check a file using stream
      * @param file the file
      * @param cfgname  cfg name
-     * @param createTokenList a function to create the simplecpp::TokenList with - throws simplecpp::Output
+     * @param createTokenList a function to create the simplecpp::TokenList with
```

Comment:
As pointed out in an earlier comment this was a wrong observation. It never throwed.